pkg/user/user.go

```diff
@@ -39,6 +47,10 @@ func (u *User) Theme() Theme {
 	return u.theme
 }
 
+func (u *User) Password() []byte {
+	return u.password
+}
+
```

Comment:
```suggestion
func (u *User) PasswordExists() bool {
	return u != nil && len(u.password) > 0
}
```

Password getter is unnecessary and weaken security. SetPassword and MatchPassword are enough. Instead, PasswordExists method can be useful in the future.

Reply:
@rot1024 we need a getter to store the password to the user doc

Reply:
ok